fairing/preprocessors/converted_notebook.py

```diff
@@ -65,16 +65,20 @@ def __init__(self,
 
         self.notebook_file = notebook_file
         self.notebook_preprocessor = notebook_preprocessor
+        self.not_overwrite = False
```

Comment:
Can you make `not_overwrite` in `__init__` with a default value? or it is intended to be a private property (in this case `_not_overwrite`)?

Reply:
I think it should be a private property. I will update it.

Reply:
Why is it a private property? If its a private property how will it get set and by whom?

Reply:
One preprocessor object only converts notebook to python script for one time. If the default path has existed, it will generate a temporary file.
`sefl._not_overwrite` checks whether it is the first time to call `preprocess()`. So, it only gets set when initialized.